test/test_grids.jl

```diff
@@ -554,7 +554,15 @@ end
         # Testing show function
         topo = (Periodic, Periodic, Periodic)
         grid = RegularRectilinearGrid(topology=topo, size=(3, 7, 9), x=(0, 1), y=(-π, π), z=(0, 2π))
-        show(grid); println();
+
+        @test try
+            show(grid)
+            true
+        catch err
+            error("error in show(::RegularRectilinearGrid)")
```

Comment:
Can you print the error message?

Reply:
Otherwise its annoying to debug because you have to reproduce the error; we can't see it in the build log for example...

Reply:
sure, but why? is it better?

Reply:
It could be

```julia
julia> f() = b
f (generic function with 1 method)

julia> f()
ERROR: UndefVarError: b not defined
Stacktrace:
 [1] f() at ./REPL[13]:1
 [2] top-level scope at REPL[14]:1

julia> a = try
       f()
       true
       catch err
       println(sprint(showerror, err))
       false
       end
UndefVarError: b not defined
false
```

Maybe in addition to a statement about where it came from (here, `show(grid)`).

Reply:
Also if we throw the error in the `catch`, the test still errors rather than fails right? It's unclear what behavior we want. I guess it's nice to know whether you got an error or a fail so we probably do want `throw(err)` in fact.